kf_lib_data_ingest/common/datafile_readers.py

```diff
@@ -76,3 +76,38 @@ def read_json_file(filepath_or_buffer, **kwargs):
     """
     kwargs["convert_dates"] = False
     return pandas.read_json(__ascii_wrap(filepath_or_buffer), **kwargs)
+
+
+def read_file(filepath_or_buffer, original_name=None, **kwargs):
```

Comment:
Maybe this module should be called `io` and we can put all of the read/write functions in here not just ones for data files. There are a few in `kf_lib_data_ingest.common.misc`

Could do in this PR or maybe for a separate PR?

Reply:
Hm. Maybe. We'd have to change the function names, because these are all for returning dataframes. There's another read_json in misc for returning a python dict.